lib/errors.js

```diff
@@ -35,7 +35,7 @@ module.exports = (translate, options) => {
       error: err,
       content: content,
       /* eslint no-process-env:0 */
-      showStack: (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'docker-compose'),
+      showStack: (options || {}).debug === true ? true : false,
```

Comment:
It's possible there is a quirk in javascript I don't understand, but I reckoning you could delete everything after the first "true" (except the comma) on this line and it should mean exactly the same thing.